internal/services/mssql/mssql_database_resource.go

```diff
@@ -1884,3 +1900,28 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
 
 	return resource
 }
+
+func calculateMaxSizeBytes(v float64) (*int64, error) {
+	var maxSizeBytes int64
+	integer := isPositiveInteger(v)
+	if integer {
```

Comment:
could we simplify this to:

```suggestion
	if isPositiveInteger(v) {
```